<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCountryMenu</class>
 <widget class="QDialog" name="CCountryMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CMyView" name="countryView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="countryLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gewähltes Basisland:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>371</y>
     <width>279</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="unitsAmountLayout">
    <item>
     <widget class="QLabel" name="unitAmountLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Gewünschte Anzahl an Einehiten:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="unitAmountSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>468</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PBLayout">
    <item>
     <widget class="QPushButton" name="tacticalMovemntPB">
      <property name="text">
       <string>Taktische Verlegung</string>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="strategicalMovemntPB">
      <property name="text">
       <string>Strategische Verlegung</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="attackPB">
      <property name="text">
       <string>Angriff!!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitPB">
      <property name="text">
       <string>Schließen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
